data-collections-exercises.md

```diff
@@ -61,37 +61,41 @@ Select all elements in `heroes` that begin with "B" using a method. [The Ruby do
 Initialize a hash called `ninja_turtle` with the properties `name`, `weapon` and `radical`. They should have values of "Michelangelo", "Nunchuks" and `true` respectively.
 
 ```rb
-# Your answer here
+ninja_turtle = { 
+                 name: "Michelangelo",
+                weapon: "Nunchuks", 
+                radical: true,
+                }  
 ```
 
 Add a key `age` to `ninja_turtle`. Set it to whatever numerical value you'd like.
 
 ```rb
-# Your answer here
+ninja_turtle.merge(age: "1000")
 ```
 
 Remove the `radical` key-value pair from `ninja_turtle`.
 
 ```rb
-# Your answer here
+ninja_turtle.delete(:radical)
 ```
 
 Add a key `pizza_toppings` to `ninja_turtle`. Set it to an array of strings (e.g., `["cheese", "pepperoni", "peppers"]`).
 
 ```rb
-# Your answer here
+ninja_turtle.merge(pizza_toppings:["cheese", "pepperoni", "peppers"])
 ```
 
 Access the first element of `pizza_toppings`.
 
 ```rb
-# Your answer here
+
```

Comment:
if used the `!` operator when you merge `ninja_turtle.merge!(pizza_toppings:["cheese", "pepperoni", "peppers"])`
you will access the pizza_toppings by using the hash symbol `ninja_turtle[:pizza_toppings]`